### PR TITLE
Class09

### PR DESCRIPTION
**Pull request by spamalope01**
Got the table to render.  Users can now add stores and see them dynamically populate the table.  Began some basic HTML organization and CSS styling (working ahead).

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,54 +1,76 @@
...
 
-Store.prototype.hourTotal = function(arr, total, avg, rand, hour) {
+Store.hourTotal = function(obj) {
   for(var j = 0; j < hours.length; j++) {
-    var monkey = Math.floor(this.getRandom(this.minCust, this.maxCust) * this.avgCookie);
-    this.totals.push(monkey);
-    total += monkey;
+    var monkey = Math.floor(obj.getRandom(obj.minCust, obj.maxCust)) * obj.avgCookie;
+    obj.totals.push(monkey);
+    obj.grandTotal += monkey;
```
Comment (sjschmidt44) [inline app.js:36]: Although it helps to remember that your variable names can be anything, be sure to double check that you're naming is appropriate. :-)

File: app.js [hunk 2]
```diff
@@ -1,54 +1,76 @@
...
-  var ulEl = document.createElement('ul');
-  pEl.textContent = this.storeName;
-
-  for (var i = 0; i < hours.length; i++) {
-    var liEl = document.createElement('li');
-    liEl.textContent = hours[i] + this.totals[i];
-    ulEl.appendChild(liEl);
+(Store.render = function() {
+  tableView = document.createElement('table');
+  tableView.setAttribute('id', 'theGrid');
```
Comment (sjschmidt44) [inline app.js:42]: We talked about this in class yesterday, and I know you saw the difference, but you can also do the same thing by `tableView.id = 'theGrid';`

File: app.js [hunk 3]
```diff
@@ -1,54 +1,76 @@
...
+var stores = [];
+var tableView;
 
-Store.prototype.totals = [];
-Store.prototype.grandTotal = 0;
+// Store.prototype.totals = [];
+// Store.prototype.grandTotal = 0;
 
 function Store (storeName, minCust, maxCust, avgCookie, id) {
   this.storeName = storeName;
```
Comment (sjschmidt44) [inline app.js:18]: You're declaring in an `id` parameter in your constructor, but not using the `id` anywhere.

File: app.js [hunk 4]
```diff
@@ -1,54 +1,76 @@
...
+    storeData.textContent = Math.floor(store.totals[hour]);
+    storeRows.appendChild(storeData);
+  }
+  var storeRowsTotals = document.createElement('th');
+  storeRowsTotals.textContent = Math.floor(store.grandTotal);
+  storeRows.appendChild(storeRowsTotals);
+  tbl.appendChild(storeRows);
 };
 
 var pikeStore = new Store('Pike Place', 17, 88, 5.2, 'pikeplace');
```
Comment (sjschmidt44) [inline app.js:76]: Likewise with the comment in your constructor, the `pikeplace` id is unused.

File: index.html [hunk 5]
```diff
@@ -2,32 +2,54 @@
...
-      <p id = "pikeplace"></p>
-      <p id = "seaTacStore"></p>
-      <p id = "southCtrStore"></p>
-      <p id = "bellSqStore"></p>
-      <p id = "alki_Store"></p>
+    <section id='greeting'>
+      <h2>Welcome to Pat's!</h2>
+      <p id="intro">
+        We're so glad you're here! Our goal is to provide you with the greatest fish based cookies anywhere in the world.  We started three years ago with just a mound of salmon, some batter and a dream. A dream that one day little people just like yourself would swim upstream to indulge in our fishy goodness!
+        <br />
```
Comment (sjschmidt44) [inline index.html:26]: `<br>` tags are considered styling, which should be done in your CSS.

File: index.html [hunk 6]
```diff
@@ -2,32 +2,54 @@
...
+      <h2>Our Featured Cookie</h2>
+      <img src="" alt="" />
+      <p>
+        <span id='randall'>The Randall</span>
+        - Named after our uncle Randall who back in the fall of 1978 rose to fame as the Upper South Northwest Regional Champion of Salmon punching.  The guy had a hell of a right cross that just sent the fish a'flyin!  And this cookie will do the same to you! Feel like you've been smacked in the gob by our uncle Randall!
+      </p>
+      <input type="button" name="name" value="Give 'em to me, yo!'">
     <script src="app.js" type="text/javascript"></script>
   </body>
+  <footer>
```
Comment (sjschmidt44) [inline index.html:42]: `<footer>` element should be inside the `<body>`

File: admin.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Pat's Salmon Cookies - Admin</title>
+    <link rel="stylesheet" href="styles.css" media="screen" title="no title" charset="utf-8">
+  </head>
+  <header>
```
Comment (sjschmidt44) [inline admin.html:8]: `<header>` and `<footer>` should both be inside the `<body>`

File: app.js (hunk 3 above)
Comment (spamalope01) [inline app.js:18]: We had to add that id....i don't remember why but it was something you suggested....so in my instances i'm adding in - after the 5.2 - the id...  However right now I honestly can't remember why that was added.

**Pull request merged**